cmd/oras/blob/fetch.go

```diff
@@ -42,7 +42,7 @@ type fetchBlobOptions struct {
 func fetchCmd() *cobra.Command {
 	var opts fetchBlobOptions
 	cmd := &cobra.Command{
-		Use:   "fetch [flags] <name@digest>",
+		Use:   "fetch [flags] {--output <file>|--descriptor} <name>@<digest>",
```

Comment:
Should we add spaces around `|`? I'm just wondering.
```suggestion
		Use:   "fetch [flags] {--output <file> | --descriptor} <name>@<digest>",
```

Reply:
Feel free to defend it.

Reply:
We should. The reason is simple: it looks better.